storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.8.rst

```diff
@@ -0,0 +1,48 @@
+==========================
+|Percona XtraBackup| 2.3.8
+==========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.8 on
+April 17th 2017. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.3.8/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3
+series.
+
+New Features
+============
+
+ |Percona XtraBackup| now uses hardware accelerated implementation of ``crc32``
+ where it is supported.
+
+ |Percona XtraBackup| has implemented new options:
+ :option:`xtrabackup --tables-exclude` and
+ :option:`xtrabackup --databases-exclude`
+ that work similar to :option:`xtrabackup --tables` and
+ :option:`xtrabackup --databases`
+ options, but exclude given names/paths from backup.
+
+ :ref:`xbstream_binary` now supports parallel extraction with the
+ ``--parallel`` option.
+
+ :ref:`xbstream_binary` now supports following new options: ``--decrypt``,
+ ``--encrypt-threads``, ``--encrypt-key``, and ``--encrypt-key-file``.
+ When ``--decrypt`` option is specified |xbstream| will automatically decrypt
+ encrypted files when extracting input stream. Either ``--encrypt-key`` or
+ ``--encrypt-key-file`` options must be specified to provide encryption key,
+ but not both. Option ``--encrypt-threads`` specifies the number of worker
+ threads doing the encryption, default is ``1``.
+
+Bugs fixed
+==========
+
+ |MySQL| 5.6 would fail to start on the incremental backup and non-default
```

Comment:
xtrabackup would not create fresh InnoDB redo logs when preparing incremental backup